eng/Version.Details.xml

```diff
@@ -123,11 +123,15 @@
     </Dependency>
     <Dependency Name="NuGet.Build.Tasks" Version="6.3.4-rc.2">
       <Uri>https://dev.azure.com/devdiv/DevDiv/_git/NuGet-NuGet.Client-Trusted</Uri>
-      <Sha>5e444458a8c4a2dac9806fc1d387b7f157669540</Sha>
+      <Sha>5ec7c4dd0086f968a8bc5a20d5cc77b1454469f3</Sha>
```

Comment:
How did the SHA change without a version change?

Reply:
To update this, I ran a darc update dependencies for the correct barID. Searching using darc, the new commit sha is the correct one. That means whenever this was originally updated for the servicing update, the sha was not correctly updated.

Looking at history, it looks like the nuget folks manually updated the version rather than running darc: https://github.com/dotnet/sdk/commit/50e815c1eba22162835757cfec0c299f2eb614c9

So this update is correct.